compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -476,6 +480,30 @@ tests damlc = testGroup "Packaging"
                   ]
               exitFailure
 
+-- | Test that a package build with --target=version never has a dependency on a package with version > version
```

Comment:
I think "version" is used with two different meanings here and we hence end up with `version > version`.

Reply:
Clearly we need to start versioning the word `version` to avoid this.